### DISCUSSION TITLE
Subpar responses when using pplx api when compared to perplexity labs

### DISCUSSION THREAD

**Discussion by optimusprime123x** [perplexityai/api-discussion#29] [General]
I tried a few questions which needed knowledge of recent events (2-3 months back), and the api response claims to have never head of it while in labs I get the desired response everytime. I tried both sonar large online and sonar small online.. same thing.

**Comment by alpineinsights** [comment]
Exactly. Same issue here. API responses are far off from what you get using Pro, closer to a Serp/Google search.

**Comment by shubhang98** [comment]
This might be useful: https://docs.perplexity.ai/faq/faq#why-are-the-results-from-the-api-different-from-the-ui

**Reply by shubhang98** [reply]
for controlling time frame of the results use the recency filter in the API.
